pubspec.yaml

```diff
@@ -12,6 +12,7 @@ dependencies:
     sdk: flutter
   gesture_x_detector: ^1.0.0
   vector_math: ^2.1.0
+  blur: ^3.0.0
```

Comment:
Do we really need this package? Isn't it possible to blur with flutter directly?